sklearn/metrics/_regression.py

```diff
@@ -534,6 +597,14 @@ def r2_score(y_true, y_pred, *, sample_weight=None,
         .. versionchanged:: 0.19
             Default value of multioutput is 'uniform_average'.
 
+    fix_when_y_true_is_constant : boolean, optional
+        Flag indicating if ``NaN`` and ``-Inf`` scores resulting from constant
+        data should be replaced with real numbers (``1.0`` if prediction is
+        perfect, ``0.0`` otherwise). Default is ``True``, a convenient settings
+        for model optimization procedures (e.g. grid search cross-validation).
+
+        .. versionadded:: 0.24
```

Comment:
```suggestion
    fix_when_y_true_is_constant : boolean, optional
        Flag indicating if ``NaN`` and ``-Inf`` scores resulting from constant
        data should be replaced with real numbers (``1.0`` if prediction is
        perfect, ``0.0`` otherwise). Default is ``True``, a convenient settings
        for model optimization procedures (e.g. grid search cross-validation).

        .. versionadded:: 1.1.0
```